.github/workflows/lmic_performance.yml

```diff
@@ -15,42 +15,47 @@ jobs:
   create-runners:
```

Comment:
all of the runner should already have many permission to the serivce. Which specific one you are looking for?

Reply:
Cloudwatch put metric data permission